happy/plan.sh

```diff
@@ -18,7 +18,9 @@ pkg_deps=(
 
 pkg_build_deps=(
   core/cabal-install
+  core/diffutils
   core/ghc
+  core/make
```

Comment:
Could you also add `core/sed` to the build deps?   I see it used below, so it would be good to declare it here, rather than rely on the studio providing it.

Reply:
Done